.github/workflows/build.yml

```diff
@@ -4,8 +4,14 @@ on:
     branches:
       - 'main'
       - 'release/*'
+    paths-ignore:
+      - 'proposals/**'
+      - 'README.md'
```

Comment:
The README does in fact change the pack output. Probably best to run CI for this.

Reply:
Oh... um, does it really affect anything?

Reply:
Nothing we check for, but the README is bundled in every NuGet package and is fed into MSBuild (see [here](https://github.com/dotnet/Silk.NET/blob/main/build/props/common.props#L46))